### PR TITLE
Fix for #17908. Added -k parameter to curl command

### PR DESCRIPTION
**Pull request by gireeshp**
This is the fix for #17908

### CODE REVIEW

File: content/en/docs/concepts/services-networking/connect-applications-service.md [hunk 1]
```diff
@@ -338,7 +338,7 @@ NAME                               READY     STATUS    RESTARTS   AGE
 curl-deployment-1515033274-1410r   1/1       Running   0          1m
 ```
 ```shell
-kubectl exec curl-deployment-1515033274-1410r -- curl https://my-nginx --cacert /etc/nginx/ssl/nginx.crt
+kubectl exec curl-deployment-1515033274-1410r -- curl -k https://my-nginx --cacert /etc/nginx/ssl/nginx.crt
```
Comment (sftim) [inline content/en/docs/concepts/services-networking/connect-applications-service.md]: This looks like the wrong fix. Why doesn't `/etc/nginx/ssl/nginx.crt` contain a CA that `curl` considers valid?

Comment (sftim) [reply content/en/docs/concepts/services-networking/connect-applications-service.md]: How about migrating this to a Task and fixing it for readability at the same time? (see #15560 
I can help out with advice.

**Comment by Rajakavitha1** [comment]
@gireeshp could you please make the changes that  @sftim suggests!!!
/hold

**Comment by gireeshp** [comment]
@Rajakavitha1 I would like to. Just not sure, how soon I can work on it. Will update here once I start. Let's close this PR anyway, without merging and keep the issue #17908 open.

**Pull request closed without merging**